src/zinc/catalog.py

```diff
@@ -284,6 +284,7 @@ def _read(self, rel_path):
         return None
 
     def _write(self, subpath: str, bytes: bytes, raw: bool = True, gzip: bool = True, max_age: Optional[int] = None):
+        log.debug(f'ZincCatalog: _write() called. (subpath: {subpath}, raw: {raw}, gzip: {gzip})')
```

Comment:
These logs were helpful when I was trying to debug the deploys and make sure they were doing what I thought they were doing.  💁‍♂️🖨️ There are already a few `log.debug`s in the Zinc codebase, so I figured I would leave them in rather than removing them.  Since they're `log.debug` and not `log.info`, they'll only appear when we have the `ACTIONS_RUNNER_DEBUG` secret set to `true` in the repository.  🤐  But if you think they're excessive, I'd be happy to remove some/all of them.  👐